server/immich-openapi-specs.json

```diff
@@ -1672,20 +1664,6 @@
         ],
         "responses": {
           "200": {
-            "content": {
```

Comment:
Is your `immich-openapi-specs.json` different than mine? The changes in the branch are getting undone here. I just ran `make api` locally with this branch up-to-date and it worked.

Reply:
No I think this is the correct spec, because you remove the `@ApiOkResponse` so these values should get removed, right?